### ISSUE TITLE
How to see the alt text of an image?

### ISSUE THREAD

**Issue by fleetingbytes** [woheller69/browser#15]
I can't find how to reveal the alt text of an image. Is there something I have missed or is it not implemented yet?

**Comment by woheller69** [comment]
if images are switched off (and connection is set as metered) images are not loaded, instead you see a text if there is one

**Comment by fleetingbytes** [comment]
Thanks, that will do for now, although I was expecting something more suitable for comic strips like XKCD or Stonetoss where they use alt text for an additional punchline.

**Comment by woheller69** [comment]
more is not possible in WebView to my knowledge

**Comment by fleetingbytes** [comment]
I also see that I can find the alt text in tge page's source. Nice feature I've been missing in so many mobile browsers!
